### DISCUSSION TITLE
No cable trace available after upgrade to 2.10

### DISCUSSION THREAD

**Discussion by kr3ator** [netbox-community/netbox#5712] [Q&A] [answered]
### Environment
* Python version: 3.9.1
* NetBox version: 2.10.4

### Steps to Reproduce
1. Have existing cables in the database with Netbox version lower than 2.10
2. Upgrade Netbox to >= 2.10
3. Go to Device --> Interfaces
4. Click the Trace button for an interface with cable conntected

### Expected Behavior
'Connection' column populated with information about the device on the other side of the cable.
After clicking the Trace button a cable trace is shown with termination A&B names, interface names, etc.

### Observed Behavior
The Connection column shows '-'.
After clicking the Trace button an almost empty page is displayed. In the box named 'Related paths'  it says 'None found'.

When I add a new cable everything works as expected.

**Comment by LuPo** [comment, marked as answer]
Have you run the migrations as per the documentation?
I suppose you are running the dockerized netbox, because the update.sh already does what you need.
If I am correct, follow the missing step on  https://github.com/netbox-community/netbox-docker/releases/tag/0.27.0
`docker-compose run --rm netbox ./manage.py trace_paths` should solve it

**Reply by sol1-matt** [reply]
we had this problem too
update.sh was run during the upgrade. I'll double check which update.sh was run.
manage.py trace_paths wouldn't have been run independent of that
will report back

**Reply by sol1-matt** [reply]
this worked for us, can't find the exact upgrade script that was run so I'll assume we did something wrong
